Libraries/oneMKL/black_scholes/black_scholes.cpp

```diff
@@ -27,6 +27,7 @@ using namespace oneapi;
 
 #include "input_generator.hpp"
 #include "black_scholes.hpp"
+#include "code_wrapper.tpp"
```

Comment:
typo? tpp => hpp

Reply:
It's not uncommon to use the tpp extension for template headers, especially when it contains template meta programs. I'm unaware of any guidelines, but don't have any strong preference.